examples/morpheusvm/controller/action_indexer.go

```diff
@@ -0,0 +1,28 @@
+// Copyright (C) 2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package controller
+
+import (
+	"context"
+
+	"github.com/ava-labs/hypersdk/chain"
+	"github.com/ava-labs/hypersdk/examples/morpheusvm/actions"
+	"github.com/ava-labs/hypersdk/extensions/indexer"
+)
+
+var _ indexer.SuccessfulTxIndexer = &successfulTxIndexer{}
+
+type successfulTxIndexer struct {
```

Comment:
We could rename it as well. We could more accurately call it an `AcceptedSubscriber`.

This would be more accurate to exactly what it's doing and an indexer would be one thing you could build as an `AcceptedSubscriber`. Similarly, you could pass in an `acceptedActionHandler` or similar, which would be just this for the MorpheusVM and would also maintain the in-memory order book in the case of TokenVM.